Manager/Manager.php

```diff
@@ -74,6 +74,14 @@ public function find($entityClass, $id, array $options = [])
     /**
      * {@inheritdoc}
      */
+    public function findAll($entityClass)
+    {
+        return $this->em->getRepository($entityClass)->findAll();
```

Comment:
This method can be a memory killer if we have too much entities.
I can accept it as a first shot, but we should think about using a cursor instead.

Reply:
Completely agree, I'm not sure how hard it is to implement a cursor. Do you perhaps know of an simple example in the current system I could check out?

Reply:
As i said before, let's keep it like it for the first shot, because the volume issue on custom entity is an edge case and open an issue to add the use of a cursor in the backlog

Reply:
sounds good